.gitignore

```diff
@@ -14,10 +14,13 @@ node_modules/
 .env.test.local
 .env.production.local
 
+
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
 
 #TODO add .env to your gitignore
+.env
```

Comment:
Just noticed this: I don't know that this will ignore the correct .env file, because it's path is `server/.env`.  Can you fix this in this branch @Vaibhav-Kambli ?

Reply:
Yes I will add the ignore route for this